decidim-core/spec/forms/notifications_settings_form_spec.rb

```diff
@@ -198,7 +200,8 @@ module Decidim
 
       context "when the notifications requirements are not met" do
         before do
-          Rails.application.secrets[:vapid] = { enabled: false }
+          allow(ENV).to receive(:fetch).and_call_original
+          allow(ENV).to receive(:fetch).with("VAPID_PUBLIC_KEY", nil).and_return(nil)
```

Comment:
And here too, we should do `Decidim.vapid_public_key`

Reply:
Fixed in 22115a5511